src/api/app/models/binary_release.rb

```diff
@@ -202,9 +202,11 @@ def reset_cache
   end
 
   def indentical_to?(binary_hash)
+    time = Time.strptime(binary_hash['buildtime'].to_s, '%s') if binary_hash['buildtime'].present?
+
     binary_disturl == binary_hash['disturl'] &&
       binary_supportstatus == binary_hash['supportstatus'] &&
-      binary_buildtime == Time.strptime(binary_hash['buildtime'].to_s, '%s')
+      binary_buildtime == time
```

Comment:
what value of 'time' is used if the hash is empty? as there is no assignment the variable is not there, so does it take the function?

Reply:
of course it does! If both times are nil then the hashes are also equal. We have to take this case into account. :nerd_face: